samplers/parser.go

```diff
@@ -479,20 +477,12 @@ func ParseEvent(packet []byte) (*UDPEvent, error) {
 	return ret, nil
 }
 
-// UDPServiceCheck is a representation of the service check.
-type UDPServiceCheck struct {
-	Name      string   `json:"check"`
-	Status    int      `json:"status"`
-	Hostname  string   `json:"host_name"`
-	Timestamp int64    `json:"timestamp,omitempty"` // represented as a unix epoch
-	Tags      []string `json:"tags,omitempty"`
-	Message   string   `json:"message,omitempty"`
-}
-
 // ParseServiceCheck parses a packet that represents a UDPServiceCheck.
```

Comment:
ditto on the docstring & the request for documenting what the SSF fields do! (:

Reply:
I think this could probably live in the ssf/ dir, as some sort of markdown doc? Possible to do that as a follow-up, but we should definitely at least outline what the fields on the things returned from this function do (: